src/clj/game/cards-ice.clj

```diff
@@ -825,6 +825,27 @@
                  :cost [:credit 1] :effect (effect (add-prop target :advance-counter 1))}]
     :subroutines [(tag-trace 2)]}
 
+   "Mausolus"
+   {:advanceable :always
+    :subroutines [{:label "Gain 1 (Gain 3)"
+                   :msg (msg "gain " (if (< (+ (:advance-counter card 0) (:extra-advance-counter card 0)) 3) 1 3) " [Credit]")
+                   :effect (req (if (< (+ (:advance-counter card 0) (:extra-advance-counter card 0)) 3)
+                                  (gain state side :credit 1)
+                                  (gain state side :credit 3)))}
+                  {:label "Do 1 net damage (Do 3 net damage)"
+                   :delayed-completion true
+                   :msg (msg "do " (if (< (+ (:advance-counter card 0) (:extra-advance-counter card 0)) 3) 1 3) " net damage")
+                   :effect (req (if (< (+ (:advance-counter card 0) (:extra-advance-counter card 0)) 3)
+                                  (damage state side eid :net 1 {:card card})
+                                  (damage state side eid :net 3 {:card card})))}
+                  {:label "Give the Runner 1 tag (and end the run)"
+                   :delayed-completion true
+                   :msg (msg "give the Runner 1 tag"
+                          (if (< (+ (:advance-counter card 0) (:extra-advance-counter card 0)) 3) "" " and end the run"))
```

Comment:
Prefer `when` for conditional statements with no "else" case. `(when (>= (+ ...) 3) " and end the run")`.